src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/NavMenu.razor.cs

```diff
@@ -19,7 +19,7 @@ public partial class NavMenu
                      new BitNavLinkItem { Name= "Label", Key = "Label",  Url = "/components/labels" },
                      new BitNavLinkItem { Name= "Link", Key = "Link", Url = "/components/links" },
                      new BitNavLinkItem { Name= "Rating", Key = "Rating", Url = "/components/rating" },
-                     new BitNavLinkItem { Name= "SearchBox", Key = "SearchBox", Url = "#" },
+                     new BitNavLinkItem { Name= "SearchBox", Key = "SearchBox", Url = "/components/searchboxes" },
```

Comment:
why the plural form of searchbox in the url?

Reply:
it seems to me, it demonstrates multiple search boxes, just like other components such as links, labels and so on

Reply:
there is only 1 "BitSearchBox" component, and the page URL needs to demonstrate that, like the FluentUI demo page:
![image](https://user-images.githubusercontent.com/6169846/125170663-393c6b80-e1c5-11eb-9d69-3d839606f11e.png)
![image](https://user-images.githubusercontent.com/6169846/125170669-3ccff280-e1c5-11eb-842d-7d8ee0a5fbd6.png)
![image](https://user-images.githubusercontent.com/6169846/125170674-3fcae300-e1c5-11eb-89b7-d9940846fa55.png)
please change it to singular format.